### DISCUSSION TITLE
Welcome to space-portfolio Discussions!

### DISCUSSION THREAD

**Discussion by sanidhyy** [sanidhyy/space-portfolio#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cornel-pe** [comment]
Here, there is no certificate part and work history part
I mean which university graduated and what certificated received.

**Reply by sanidhyy** [reply]
Feel free to add those details yourself. I didn't include them because I didn't see the need for them in this project.

**Reply by cornel-pe** [reply]
sure, anyway, your project is awesome, it's very necessary for many people who don't have enough skill in frontend.
I will keep looking on your future projects.
regards.

**Reply by sanidhyy** [reply]
Thanks. I will think to add those features in future when I will get some time.
Regards.
